### ISSUE TITLE
Create a "use example data" toggle for the app

### ISSUE THREAD

**Issue by eneskemalergin** [LangeLab/DataAnalysisPipeline#22]
At the current state, the user needs to go to GitHub and download the data which can be around 60MB. Instead of download and upload, we can simply include a checklist element to opt for using example data directly on the app.

**Comment by phegnal** [comment]
Yes please. One of the things I had hoped would get implemented before Siyuan had to leave. 
In addition we should make a stripped down version (header + couple rows) of the files available for download as reference for formatting
